README.rst

```diff
@@ -224,7 +224,7 @@ Executing custom commands
 -------------------------
 
 For experimenting it can be useful to execute arbitrary commands against the endpoint.
-You can access the available methods by calling ``songpal list-all``.
+You can access the available methods by calling ``songpal list_all``.
```

Comment:
The dash is correct here (with click 7.0 onwards, see https://github.com/pallets/click/issues/1123), maybe this requirement should be pinned so that people will upgrade their click installations. Please remove this.

Reply:
Is there a reason not to put this into requirements.txt as `click>=7.0`?

Reply:
Done!